grpc/rpc/bmc_test.go

```diff
@@ -168,6 +168,64 @@ func TestReset(t *testing.T) {
 	}
 }
 
+func newDeactivateSOLRequest(authErr bool) *v1.DeactivateSOLRequest {
+	var auth *v1.DirectAuthn
+	if authErr {
+		auth = &v1.DirectAuthn{
+			Host: &v1.Host{
+				Host: "",
+			},
+			Username: "ADMIN",
+			Password: "ADMIN",
+		}
+	} else {
+		auth = &v1.DirectAuthn{
+			Host: &v1.Host{
+				Host: "127.0.0.1",
+			},
+			Username: "ADMIN",
+			Password: "ADMIN",
+		}
+	}
+	return &v1.DeactivateSOLRequest{
+		Authn: &v1.Authn{
+			Authn: &v1.Authn_DirectAuthn{
+				DirectAuthn: auth,
+			},
+		},
+		Vendor: &v1.Vendor{
+			Name: "local",
+		},
+	}
+}
+
+func TestDeactivateSOL(t *testing.T) {
+	testCases := []struct {
+		name        string
+		expectedErr error
+		in          *v1.DeactivateSOLRequest
+		out         *v1.DeactivateSOLResponse
+	}{
+		{"success", nil, newDeactivateSOLRequest(false), &v1.DeactivateSOLResponse{TaskId: ""}},
+		{"missing auth err", errors.New("input arguments are invalid: invalid field Authn.DirectAuthn.Host.Host: value '' must not be an empty string"), newDeactivateSOLRequest(true), nil},
+	}
+
+	for _, tc := range testCases {
+		tc := tc
```

Comment:
any reason to copy this?

Reply:
Copy/paste tagalong from existing similar functions -- it does look nonsensical, and AFAICT removing it doesn't seem to introduce any problems, so probably safe to drop I suppose...